var/spack/repos/builtin/packages/pygmo/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```

Comment:
Why is there an empty `py-ytopt/package.py` file?

Reply:
not sure if I understand this. I don't see any empty package. Maybe it's related to the file permission, will correct that.

Reply:
Yes, looks like it is due to file permissions.